python/py-kivy/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-kivy
-version             1.10.0
-revision            1
+version             1.10.1
+revision            0
```

Comment:
We do not start revision with "0". That is, if revision would be zero, we leave the line out entirely.

Reply:
@pmetzger Thanks for the review

Ok i'll fix 

Regards,